guides/common/modules/ref_statements-in-managed-dhcpv4.adoc

```diff
@@ -1,39 +1,39 @@
-[id="DHCP_Options_{context}"]
-= {Project} and DHCP options
+[id="statements-in-managed-dhcpv4"]
+= Statements in managed DHCPv4
```

Comment:
I think we should keep using `option` because DHCP has options, not statements.

I can't quite figure out if ISC dhcpd maps `next-server` to option 66 (https://www.rfc-editor.org/rfc/rfc2132#section-9.4) or option 150 (https://www.rfc-editor.org/rfc/rfc5859). Reading the source I get the impression it uses option 17 for `next-server`, but that's the root path according to https://www.rfc-editor.org/rfc/rfc2132.html#section-3.19.

Regardless of what it exactly maps to, it's still an option and not a statement.